__tests__/app.test.js

```diff
@@ -215,7 +215,47 @@ describe('GET/api/articles/:article_id/comments', () => {
     })
 })
 
-
+describe('POST/api/articles/:article_id/comments', () => {
+    test('200: should add a comment to an article (specified by article_id) and respond with the posted comment', () => {
+        const testComment = {
+            username: 'butter_bridge',
+            body: 'test comment'
+        }
+        return request(app)
+        .post('/api/articles/2/comments')
+        .send(testComment)
+        .expect(201)
+        .then((response) => {
+            expect(response.body.comment).toBe('test comment')
```

Comment:
This is good, however, you should be expecting to receive the whole comment object and not just the `body` property